code/modules/clothing/masks/gasmask.dm

```diff
@@ -159,7 +159,7 @@
 	magical = TRUE
 
 /obj/item/clothing/mask/gas/mime/nodrop
-	flags = NODROP
+	flags = NODROP | BLOCK_GAS_SMOKE_EFFECT | AIRTIGHT
```

Comment:
The normal ones don't have these flags. Calling this path nodrop won't convey the true changes this version has compared to the default one. Default one only has `FLAMMABLE`.

Reply:
That's weird. I must have assumed the mime mask had the same flags as the clown mask. A little strange that it doesn't, actually. Iirc AIRTIGHT is required for internals, so AIRTIGHT should probably stay on regardless to avoid any vox or plasmaman shenanigans. Will make the changes when I have a moment.